app/controllers/provider_interface/application_choices_controller.rb

```diff
@@ -9,21 +11,13 @@ def index
       end
     end
 
-    def show
-      application_choice = GetApplicationChoicesForProvider.call(provider: current_user.provider)
-        .find(params[:application_choice_id])
-
-      @application_choice = ApplicationChoicePresenter.new(application_choice)
-    end
+    def show; end
 
   private
 
-    # Stub out the current user and their organisation. Will be replaced
-    # by a proper ProviderUser when implementing Signin.
-    def current_user
-      fake_user_class = Struct.new(:provider)
-      fake_provider_class = Struct.new(:code)
-      fake_user_class.new(fake_provider_class.new('ABC'))
+    def set_application_choice
```

Comment:
This should set an instance of `ApplicationChoicePresenter`.

Reply:
fixed